ui/admin/app/components/auth-and-model-providers/ProviderConfigure.tsx

```diff
@@ -174,18 +173,9 @@ export function ProviderConfigureContent({
 				</DialogTitle>
 			</DialogHeader>
 
-			{(provider.id === CommonModelProviderIds.ANTHROPIC ||
-				provider.id == CommonModelProviderIds.ANTHROPIC_BEDROCK) && (
+			{provider.description && (
 				<DialogDescription className="px-4">
-					Note: Anthropic does not have an embeddings model and{" "}
-					<Link
-						target="_blank"
-						rel="noreferrer"
-						to="https://docs.anthropic.com/en/docs/build-with-claude/embeddings"
-					>
-						recommends
-					</Link>{" "}
-					Voyage AI.
+					<ReactMarkdown>{provider.description ?? ""}</ReactMarkdown>
```

Comment:
nit: because you prefixed this with `{provider.description && (...`, TS will know that `provider.description` will never be nullish

```suggestion
					<ReactMarkdown>{provider.description}</ReactMarkdown>
```